backend/v2/epochs/dependencies.py

```diff
@@ -61,7 +56,48 @@ def get_epochs_subgraph(
     return EpochsSubgraph(settings.subgraph_endpoint)
 
 
+async def get_current_epoch(epochs_contracts: GetEpochsContracts) -> int:
+    return await epochs_contracts.get_current_epoch()
```

Comment:
This seems to be just a pass-thru so maybe we can remove it?
Also if we want to keep this we should align the get_current_epoch_v1 endpoint

Reply:
if you see this custom i'd suggest to keep it in the deps and i changed get_current_epoch_v1 as well; it makes our views more DI-able and neat instead of calling epochs_contracts.get_current_epoch directly either in a dependency or in a router body, wdyt?